ckanext/canada/metadata_xform.py

```diff
@@ -116,7 +121,10 @@ def _process(line, portal=False):
         rec['spatial_representation_type'] = []
 
     if not rec.get('maintainer_email'):
-        rec['maintainer_email'] = 'open-ouvert@tbs-sct.gc.ca'
+        if rec['collection'] == u'geogratis':
+            rec['maintainer_email'] = 'NRCan.geogratis-geogratis.RNCan@canada.ca'
+        else:
+            rec['maintainer_email'] = 'open-ouvert@tbs-sct.g.ca'
```

Comment:
better yet, remove the maintainer_email case for non-geogratis because that gets filled in by the maintainer_email validator when left blank